app/presenters/scanned_resource_show_presenter.rb

```diff
@@ -7,4 +7,11 @@ def file_presenter_class
   def pending_uploads
     @pending_uploads ||= PendingUpload.where(curation_concern_id: id)
   end
+
+  def parent
+    ids = solr_document.fetch('ordered_by_ssim', [])
+    CurationConcerns::PresenterFactory.build_presenters(ids[0...1],
```

Comment:
That is definitely a problem. The link to the scanned resource from the multi-volume work could be something like this: ```/concern/container/:parent_id/scanned_resources/:id```